wait-serv.py

```diff
@@ -0,0 +1,3 @@
+from waitress import serve
+import kkbridge
+serve(kkbridge.create_app(), host='0.0.0.0', port=1646)
```

Comment:
pretty sure this is world-accessible?
```suggestion
serve(kkbridge.create_app(), host='127.0.0.1', port=1646)
```